CONTRIBUTORS.txt

```diff
@@ -164,3 +164,41 @@ Order doesn't matter (not that much, at least ;)
 * Jason Owen: contributor
 
 * Mitchell Young: minor adjustment to docparams
+
+* Marianna Polatoglou: minor contribution for wildcard import check
```

Comment:
Can we remove these `CONTRIBUTORS` entries? I think most of them appeared only in 2.X so I'm not sure it makes sense to backport them to 1.9.

Reply:
Fixed